### PR TITLE
feat: add gitpod files

### PR DESCRIPTION
**Pull request by adschwartz**
This PR adds files needed for a fully working gitpod integration.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - name: Setup Kurtosis
+    command: |
+      echo "deb [trusted=yes] https://apt.fury.io/kurtosis-tech/ /" | sudo tee /etc/apt/sources.list.d/kurtosis.list
+      sudo apt update
+      sudo apt install kurtosis-cli
+      kurtosis run github.com/fake-package/fake-package-purely-for-analytics || kurtosis clean -a
```
Comment (adschwartz) [inline .gitpod.yml:10]: @h4ck3rk3y should we disable analytics in order to not pollute metrics?

Comment (h4ck3rk3y) [reply .gitpod.yml:10]: this allows us to track how people are using GitPod

File: .gitpod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1 @@
+FROM gitpod/workspace-base
```
Comment (h4ck3rk3y) [inline .gitpod.Dockerfile]: new line

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    port: 9710
+    onOpen: ignore
+  - name: Kurtosis Engine Grpc Proxy Listen
+    port: 9710
+    onOpen: ignore
+
+
+vscode:
+  extensions:
+    - Kurtosis.kurtosis-extension
```
Comment (h4ck3rk3y) [inline .gitpod.yml]: new line

File: README.md [hunk 4]
```diff
@@ -11,9 +11,7 @@ If you have [Kurtosis installed][install-kurtosis], run:
 kurtosis run github.com/YOURUSER/THISREPO
 ```
 
-<!-- TODO Add a URL-encoded version of github.com/YOURUSER/THISREPO to right after "KURTOSIS_PACKAGE_LOCATOR=" in the link below -->
-<!-- TODO You can URL-encode a string using https://www.urlencoder.org/ -->
-If you don't have Kurtosis installed, [click here to run this package on the Kurtosis playground](https://gitpod.io/#KURTOSIS_PACKAGE_LOCATOR=/https://github.com/kurtosis-tech/playground-gitpod).
+If you don't have Kurtosis installed, [click here to run this package on the Kurtosis playground](https://gitpod.io/?editor=code#https://github.com/kurtosis-tech/playground-gitpod).
```
Comment (h4ck3rk3y) [inline README.md]: autoStart=true

^ this param skips the box

**Review by h4ck3rk3y — APPROVED** [review]

File: .gitpod.yml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+      code README.md
+      source <(kurtosis completion bash)
+      echo "source <(kurtosis completion bash)" >> ~/.bashrc
+      clear  
+
+ports:
+  - name: Kurtosis Engine Grpc Listen
+    port: 9710
+    onOpen: ignore
+  - name: Kurtosis Engine Grpc Proxy Listen
```
Comment (mieubrisse) [inline .gitpod.yml]: You can kill this; no longer used/needed

Comment (adschwartz) [reply .gitpod.yml]: Done

**Pull request merged**